src/main/java/duke/Messages.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+/**
+ * Stores the string messages of the application.
+ */
+public class Messages {
+
+    // Regular messages
+    public static final String GREETING_MESSAGE = " Hello! I'm ChatNat!\n What can I do for you?";
+
+    public static final String LIST_MESSAGE = " Here are the tasks in your list:";
+
+    public static final String FIND_MESSAGE = " Here are the matching tasks in your list:";
+
+    public static final String INSERT_MESSAGE = " Got it. I've added this task:\n   %s\n Now you "
+            + "have %d %s in the list.";
+    public static final String MARKED_MESSAGE = " Nice! I've marked this task as done:\n   %s";
+
+
+    public static final String UNMARKED_MESSAGE = " OK, I've marked this task as not done yet:\n "
```

Comment:
I like it that you stored the messages in one class. Would it be better if they had common prefix? (eg: MESSAGE)